modules/nw-ingress-controller-endpoint-publishing-strategies.adoc

```diff
@@ -5,4 +5,23 @@
 [id="nw-ingress-controller-endpoint-publishing-strategies_{context}"]
 = Ingress controller endpoint publishing strategy
 
+*`NodePort` endpoint publishing strategy*
```

Comment:
Should be `NodePortService` instead of `NodePort`. 
see: https://github.com/openshift/api/blob/85e16cc5dd7c18585be10a6cac2f286e2cc079f3/operator/v1/types_ingress.go#L203

Reply:
Got it, thanks!